core.d.ts

```diff
@@ -489,7 +489,7 @@ export function fileTypeStream(webStream: AnyWebReadableStream<Uint8Array>, opti
 export declare class FileTypeParser {
 	detectors: Iterable<Detector>;
 
-	constructor(options?: {customDetectors?: Iterable<Detector>});
+	constructor(options?: {customDetectors?: Iterable<Detector>; abortSignal: AbortSignal});
```

Comment:
It's usually called `signal`.

```suggestion
	constructor(options?: {customDetectors?: Iterable<Detector>; signal: AbortSignal});
```